data-serving/scripts/constants.py

```diff
@@ -1,2 +1,22 @@
 ''' Valid values for sex field. '''
 VALID_SEXES = ['female', 'male', 'other']
+
+LOCATIONS = {
+    'wuhan': {
+        'country': 'China',
+        'administrativeAreaLevel1': 'Hubei',
+        'administrativeAreaLevel2': 'Wuhan City',
+        'locality': 'Wuhan City',
+        'geometry': {
+            'latitude': 30.62506,
+            'longitude': 114.3421
+        }
+    },
+    'iran': {
+        'country': 'Iran',
```

Comment:
There's less resolution as compared to the `wuhan` entry. Is this just made to mirror what we see in the existing data?

Reply:
yep, it's because Wuhan is a locality, so we can infer locality, admin1, admin2, country, etc. 
whereas for entries where the full string is "Iran", we don't have any more detailed info than just the country (and the centroid lat/long of the country)